src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -36,6 +36,11 @@ private static bool IsCompilerGenerated(MethodDefinition methodDefinition)
             return false;
         }
 
+        private static bool IsCompilerGenerated(FieldDefinition fieldDefinition)
+        {
+            return fieldDefinition.DeclaringType.CustomAttributes.Any(ca => ca.AttributeType.FullName == typeof(CompilerGeneratedAttribute).FullName);
```

Comment:
Could be inside nested class? In this case it's more correct to search up to the outermost declaring type as done for ```IsCompilerGenerated(MethodDefinition methodDefinition)``` overload.

Reply:
I'm not sure if it's possible...do you think it's possible?

Reply:
I'll merge as is for now I'm not sure if a field for a lambda could be nested inside other generated type...I cannot find samples, if you find it let me know and we'll amend.